src/styles/navbar.scss

```
@@ -0,0 +1,114 @@
+.navbar {
+  align-items: center;
+  color: $text-gray;
+  display: flex;
+  flex-direction: row;
+  font-size: $text-base;
+  justify-content: space-between;
+  padding: $m-md;
+  position: relative;
+
+  &__logo {
+    max-height: $m-lg;
+  }
+
+  &__burger {
+    background-color: transparent;
+    border: none;
+  }
+
+  &__burger-bar {
+    background-color: $dark-purple;
+    height: 2px;
+    margin: $m-xs 0;
+    width: 1.5rem;
+  }
+
+  &__wrapper {
+    background-color: $white;
+    display: none;
+    height: 100vh;
+    left: 0;
+    padding: $m-lg 0;
+    position: absolute;
+    top: $m-xxl;
+    width: 100%;
+    z-index: 100;
+
+    &.-displayed {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+  }
+
+  &__list-item {
+    margin: $m-base 0;
+
+    &:hover {
+      color: $purple;
+      cursor: pointer;
+    }
+  }
+
+  &__button {
+    background-color: $off-white;
+    border: 1px solid $light-gray;
+    border-radius: $radius-sm;
+    color: $text-gray;
+    cursor: pointer;
+    font-weight: $bold;
+    height: $m-lg;
+    margin: $m-sm;
+    padding: 0 1.5*$m-md;
```

```suggestion
    padding: 0 $m-lg;
```

Try to avoid such calculations and just use variables

---

Applied